docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,174 @@
+Enable cross-cluster replication in Aiven for Apache Cassandra® |private beta|
+==============================================================================
+
+.. important::
+
+   This private beta feature is available on request only. Contact sales@Aiven.io or support@Aiven.io to opt in. 
+
+This article provides you with instructions on how to set up the cross-cluster replication (CCR) in your Aiven for Apache Cassandra® service using the Aiven console.
+
+About enabling CCR
+------------------
+
+You can enable CCR either when creating a new Aiven for Apache Cassandra service or on an existing service. For that purpose, you can use the following tools:
```

Comment:
yes, that makes sense. How about adding the note below for the time being, and later on, when the concept is available, adding a link inside this note leading to the relevant info in the concept?
`Note: Enabling CCR on an existing service can affect your productions workload since CCR imposes restrictions on replication settings of the user's keyspace. Make sure you understand the impact before you enable CCR on your service`